build.gradle

```diff
@@ -829,6 +832,46 @@ distributions {
     }
   }
 }
+
+ospackage {
+  packageName = distributions.main.baseName
+  version = version
+
+//  release = new Date().format('yyyyMMdd-HHmmss')
+//  arch = NOARCH
+
+  os = LINUX
+
+  maintainer = 'SnappyData, Inc.'
+  packageDescription = productName + ', the Spark Database. Stream - Transact - Analyze - Predict all in one cluster'
+  summary = productName + ' Installer'
+
+  user = 'snappydata'
+  permissionGroup = 'snappydata'
+
+  from(snappyProductDir) {
+//    fileMode = 0664
+    into '/opt/' + distributions.main.baseName
+  }
+}
+
+buildRpm {
+  requires('glibc')
+  dependsOn ':packageVSD'
+  if (rootProject.hasProperty('hadoop-provided')) {
+    classifier 'without_hadoop'
+  }
+}
+
+buildDeb {
+  requires('libc6')
```

Comment:
Also add curl as requirement for both rpm and deb (required by snappy-job.sh and AWS checks).

Reply:
Thanks @sumwale for your response(s) - I presume you will work on this internally to be able to better test this PR.